Add Testery set up to Tutorials

dacharyc opened the pull request:
Alrighty, @q0rban , here's an initial pass at adding a "Testery.io Integration" doc to the docs site. Can you take a look at the example `config.yml` and make any corrections you spot, and/or comments/fixes for the overall page?

q0rban commented on content/starter-configs/tutorials/testery.md:
This should be:

```
- apt-get install -y python3
- apt-get install -y python3-pip
```

OR

```
- apt-get install -y python3 python3-pip
```

q0rban commented on content/starter-configs/tutorials/testery.md:
The applicable config gets lost in here. What do you think about removing all of the detail above and doing something like this instead:

```yaml
services:
  php:
    commands:
      init:
        ...
        # Install pip so you can get the Testery CLI.
        - apt-get update
        - apt-get install -y python3 python3-pip
      build:
        ...
        # Check for upgrades to the Testery command line interface
        - pip3 install testery --upgrade
        # Set up Testery to run tests against the Tugboat environment, using Tugboat environment variables
        - testery update-environment --create-if-not-exists --token "$TESTERY_TOKEN" --key "${TUGBOAT_PREVIEW}" --name
          "${TUGBOAT_PREVIEW}" --variable "TUGBOAT_DEFAULT_SERVICE_URL=${TUGBOAT_DEFAULT_SERVICE_URL}"
        # Start a test run.
        - testery create-test-run --token "$TESTERY_TOKEN" --git-ref "$TUGBOAT_PREVIEW_SHA" --project "testery"
          --environment "${TUGBOAT_PREVIEW}"
```

q0rban commented on content/starter-configs/tutorials/testery.md:
Maybe we should say "Start testing your Previews!" or something? Or just remove this paragraph?

dacharyc commented:
Alrighty, I've updated the Testery doc, but I've also removed the deprecated testing code snippets we discussed yesterday in the Slack, @q0rban . I've added permanent redirects for those to the Testery page; can you verify I've done those rewrite rules correctly, @bchavet ?

q0rban commented on static/.htaccess:
```diff
@@ -92,5 +92,7 @@
     RewriteRule ^starter-configs/code-snippets/generic-lamp/?$ %{ENV:CWD}starter-configs/examples/generic-lamp/ [L,R=301]
     RewriteRule ^starter-configs/code-snippets/static-jekyll/?$ %{ENV:CWD}starter-configs/examples/static-jekyll/ [L,R=301]
+    RewriteRule ^starter-configs/code-snippets/functional-tests/?$ %{ENV:CWD}starter-configs/tutorials/testery/ [L,R=301]
+    RewriteRule ^starter-configs/code-snippets/simpletest/?$ %{ENV:CWD}starter-configs/tutorials/testery/ [L,R=301]
```
These work great. Tested on Tugboat.
